app/src/lib/ssh/ssh.ts

```diff
@@ -0,0 +1,32 @@
+import * as fse from 'fs-extra'
+import memoizeOne from 'memoize-one'
+import { enableWindowsOpenSSH } from '../feature-flag'
+import { getBoolean } from '../local-storage'
+
+const WindowsOpenSSHPath = 'C:\\Windows\\System32\\OpenSSH\\ssh.exe'
+
+export const UseWindowsOpenSSHKey: string = 'useWindowsOpenSSH'
+
+export const isWindowsOpenSSHAvailable = memoizeOne(
+  async (): Promise<boolean> => {
+    if (!__WIN32__ || !enableWindowsOpenSSH()) {
+      return false
+    }
+
+    return await fse.pathExists(WindowsOpenSSHPath)
```

Comment:
For some reason this is `true` on Windows 11 for ARM, but later `git` can't find it 🤔